Anlab.Mvc/Client/components/ViewMode.tsx

```diff
@@ -32,8 +32,10 @@ export class ViewMode extends React.Component<IViewModeProps, any> {
         );
     }
 
-    private _handleChange = () => {
-        this.props.switchView(!this.props.placingOrder);
+    private _handleChange = (doNothing: boolean) => {
+        if (!doNothing) {
+            this.props.switchView(!this.props.placingOrder);
+        }
```

Comment:
@jSylvestre how about instead something like `this.props.switchView('viewname')` instead of the do nothing.  Then it would be like how tabs or radios work.   So you'd change with this._switchView('browse') and then it would call the props.switchView which would take in the view name (or bool if you want to track it, but that limits us to binary)